src/_round.ts

```diff
@@ -0,0 +1,21 @@
+export function _round(roundingFn: (value: number) => number) {
+  return (value: number, precision: number) => {
+    if (!Number.isInteger(precision)) {
```

Comment:
We should special-case the `precision === 0` case to reduce our footprint:

```suggestion
  return (value: number, precision: number) => {
    if (precision === 0) {
      return roundingFn(value);
    }

    if (!Number.isInteger(precision)) {
```